### PR TITLE
Prepare for Melodic

### PR DESCRIPTION
**Pull request by Timple**

### CODE REVIEW

**Comment by Timple** [comment]
Maybe most of the bootstrap scripts can be merged into bootstrap-ros and the version specific ones only set the distro and then call the same script.

File: ci/install-package.sh [hunk 1]
```diff
@@ -76,9 +76,9 @@ docker exec tue-env bash -c "export CI='true'; source /home/amigo/.bashrc; tue-g
 echo -e "\e[35m\e[1m Reset package to this commit \e[0m"
 if [[ $PULL_REQUEST == "false" ]]; 
 then
-    echo -e "\e[35m\e[1m cd ~/ros/kinetic/system/src/$PACKAGE && git reset --hard $COMMIT \e[0m"
-    docker exec tue-env bash -c "cd ~/ros/kinetic/system/src/$PACKAGE && git reset --hard $COMMIT"
+    echo -e "\e[35m\e[1m cd ~/ros/$ROSDISTRO/system/src/$PACKAGE && git reset --hard $COMMIT \e[0m"
```
Comment (MatthijsBurgh) [inline ci/install-package.sh]: Where is ROSDISTRO defined?

Comment (Timple) [reply ci/install-package.sh]: Was a typo, should be ROS_DISTRO

**Comment by MatthijsBurgh** [comment]
Shit is broken: https://travis-ci.org/tue-robotics/tue_config/builds/450219085

**Comment by ar13pit** [comment]
CI issues fixed in #302

**Comment by Timple** [comment]
Whups

**Pull request merged**